doc/introduction/interfaces/tfe.rst

```diff
@@ -0,0 +1,159 @@
+.. _tf_interf:
```

Comment:
This file needs to be deleted (out of date)

Reply:
Really? But where would we explain how the interface works? What part it out of date?

Reply:
Ah, sorry, this comment wasn't very informative! I meant more that `tfe` was replaced with a `tf` module when TensorFlow 2.0 was released, and eager mode made the default UI.

This change is already in master, so I think master needs to be merged in, and the resulting conflict fixed up (defaulting to the new master version).